README.md

```diff
@@ -36,7 +36,7 @@ bmt rcstart &&
 bmt setup
 ```
 
-It is best to reboot after this just to be sure everything is applied.
+It is best to reboot after this just to be ensure things will start properly on reboot.
```

Comment:
The issue is it won't work until you won't reboot, it's not just that you want to make sure.
This is because I opted to KISS things and not include the while-running commands too.